lib/stat_tracker.rb

```diff
@@ -1,17 +1,35 @@
+require_relative "./game"
+require_relative "./team"
+require_relative "./gameteam"
+
 class StatTracker
   attr_reader :games, :teams, :game_teams
-  
+
   def initialize(games, teams, game_teams)
     @games = games
     @teams = teams
     @game_teams = game_teams
   end
-  
+
   def self.from_csv(locations)
     games = Game.create_from_csv(locations[:games])
     teams = Team.create_from_csv(locations[:teams])
     game_teams = GameTeam.create_from_csv(locations[:game_teams])
     StatTracker.new(games, teams, game_teams)
   end
 
-end
\ No newline at end of file
+  def highest_total_score
```

Comment:
I think we'll need to change this from an instance method run on a StatTracker object to one that lives within the class itself, but this is a great start to it!

Reply:
According to our stat_tracker_spec it seems the tests should be written in stat_tracker, but I may be misunderstanding.